src/content/app/species-selector/components/species-search-field/SpeciesSearchField.tsx

```diff
@@ -14,110 +14,77 @@
  * limitations under the License.
  */
 
-import React, { useEffect } from 'react';
+import React, { type FormEvent } from 'react';
+import classNames from 'classnames';
 
-import { useAppSelector, useAppDispatch } from 'src/store';
-import useSpeciesSelectorAnalytics from 'src/content/app/species-selector/hooks/useSpeciesSelectorAnalytics';
+import { useAppDispatch, useAppSelector } from 'src/store';
 
-import {
-  updateSearch,
-  setSelectedSpecies,
-  clearSelectedSearchResult,
-  clearSearch
-} from 'src/content/app/species-selector/state/speciesSelectorSlice';
+import { getSpeciesSearchQuery } from 'src/content/app/species-selector/state/species-selector-search-slice/speciesSelectorSearchSelectors';
+import { setQuery } from 'src/content/app/species-selector/state/species-selector-search-slice/speciesSelectorSearchSlice';
 
-import {
-  getSearchText,
-  getSearchResults,
-  getSelectedItemText
-} from 'src/content/app/species-selector/state/speciesSelectorSelectors';
-
-import SpeciesSearchMatch from '../species-search-match/SpeciesSearchMatch';
-import AutosuggestSearchField from 'src/shared/components/autosuggest-search-field/AutosuggestSearchField';
-
-import type {
-  SearchMatch,
-  SearchMatches
-} from 'src/content/app/species-selector/types/species-search';
-
-import { MINIMUM_SEARCH_LENGTH } from 'src/content/app/species-selector/constants/speciesSelectorConstants';
+import ShadedInput from 'src/shared/components/input/ShadedInput';
+import { PrimaryButton } from 'src/shared/components/button/Button';
 
 import styles from './SpeciesSearchField.scss';
 
-export const NOT_FOUND_TEXT = 'Sorry, we have no data for this species';
-
-export const SpeciesSearchField = () => {
-  const searchText = useAppSelector(getSearchText);
-  const matches = useAppSelector(getSearchResults);
-  const selectedItemText = useAppSelector(getSelectedItemText);
-  const dispatch = useAppDispatch();
-
-  useEffect(() => {
-    return () => clear();
-  }, []);
+export type Props = {
+  query: string;
+  onSearchSubmit: () => void;
+  canSubmit?: boolean;
+  onInput?: ((event: FormEvent<HTMLInputElement>) => void) | (() => void);
+};
 
-  const { trackAutocompleteSpeciesSelect } = useSpeciesSelectorAnalytics();
+export const SpeciesSearchField = (props: Props) => {
```

Comment:
Since we still haven't got a better name for this component was wondering how e can rename it. Would `SimpleSpeciesSearchField` make it less confusing/better for imports?

Reply:
It's fine; we have a default export vs a named export.